ocamldoc/odoc_misc.ml

```diff
@@ -217,7 +217,7 @@ let string_of_info i =
   let module M = Odoc_types in
   (match i.M.i_deprecated with
     None -> ""
-  | Some d -> Odoc_messages.deprecated^"! "^(string_of_text d)^"\n")^
+  | Some d -> Odoc_messages.deprecated^".! "^(string_of_text d)^"\n")^
```

Comment:
Does this end up in plain text output (i.e. is a typo) or is it something deeper?

Reply:
"Deprecated.!" is not found in the manual.txt output.

I don't understand what's going on here but I have not changed the net result: if this code was wrong (or right) before, it's wrong (or right) now.

Reply:
If it's not in the plain text manual (and so not really testable), then I agree with leaving it (weird as it looks!)

Reply:
This is wrong, but this is only appear on a difficult to trigger path: this is only visible on the texi output for inline record fields:

```ocaml
type t =
| A of { x:int (** A.x @deprecated "A.x" *); y :int } (** A @deprecated "A" *)
| B
```
is rendered as
```
  | A of @{
   x : int;(* Deprecated.! "A.x"
A.x
 *)
   y : int;
@}     
     (*
     Deprecated. "A"

A

 *)
  | B
```

Reply:
So I can just remove the exclamation mark?

Reply:
Indeed.